src/phoenix/experimental/evals/models/anthropic.py

```diff
@@ -0,0 +1,170 @@
+    "claude-2.0": 100000,
+    "claude-instant-1.2": 100000,
+}
+
+
+@dataclass
+class AnthropicModel(BaseEvalModel):
+    model: str = "claude-2.1"
+    """The model name to use."""
+    temperature: float = 0.0
+    """What sampling temperature to use."""
+    max_tokens: int = 256
+    """The maximum number of tokens to generate in the completion."""
+    top_p: float = 1
+    """Total probability mass of tokens to consider at each step."""
+    top_k: int = 256
+    """The cutoff where the model no longer selects the words"""
+    stop_sequences: List[str] = field(default_factory=list)
+    """If the model encounters a stop sequence, it stops generating further tokens. """
+    max_retries: int = 6
+    """Maximum number of retries to make when generating."""
+    retry_min_seconds: int = 10
+    """Minimum number of seconds to wait when retrying."""
+    retry_max_seconds: int = 60
+    """Maximum number of seconds to wait when retrying."""
+    extra_parameters: Dict[str, Any] = field(default_factory=dict)
+    """Any extra parameters to add to the request body (e.g., countPenalty for a21 models)"""
+    max_content_size: Optional[int] = None
+    """If you're using a fine-tuned model, set this to the maximum content size"""
+
+    def __post_init__(self) -> None:
+        self._init_environment()
+        self._init_client()
+        self._init_tiktoken()
+        self._init_rate_limiter()
+        self.retry = self._retry(
+            error_types=[],  # default to catching all errors
+            min_seconds=self.retry_min_seconds,
+            max_seconds=self.retry_max_seconds,
+            max_retries=self.max_retries,
+        )
+
+    def _init_environment(self) -> None:
+        try:
+            import tiktoken
+
+            self._tiktoken = tiktoken
+        except ImportError:
+            self._raise_import_error(
+                package_name="tiktoken",
+            )
+
+    def _init_client(self) -> None:
+        try:
+            import anthropic  # type:ignore
+
+            self._anthropic = anthropic
+            self.client = self._anthropic.Anthropic()
+            self.async_client = self._anthropic.AsyncAnthropic()
+        except ImportError:
+            self._raise_import_error(
+                package_name="anthropic",
+            )
+
+    def _init_tiktoken(self) -> None:
+        try:
+            encoding = self._tiktoken.encoding_for_model(self.model)
+        except KeyError:
+            logger.warning("Warning: model not found. Using cl100k_base encoding.")
+            encoding = self._tiktoken.get_encoding("cl100k_base")
+        self._tiktoken_encoding = encoding
+
+    def _init_rate_limiter(self) -> None:
+        self._rate_limiter = RateLimiter(
+            rate_limit_error=self._anthropic.RateLimitError,
+            max_rate_limit_retries=10,
+            initial_per_second_request_rate=1,
+            maximum_per_second_request_rate=20,
+            enforcement_window_minutes=1,
```

Comment:
shouldn't these just be defaults?

Reply:
yeah I think that is probably the right way to go. I am tempted to try to tune these per model based on our observed behavior of each API but perhaps it's too much given the variation between different users' rate limits

Reply:
Okay no worries - just trying to avoid magic numbers:)